pkg/cmd/server/kubernetes/master_config_test.go

```diff
@@ -109,7 +116,9 @@ func TestCMServerDefaults(t *testing.T) {
 			ClusterName:                       "kubernetes",
 			TerminatedPodGCThreshold:          12500,
 			VolumeConfiguration: componentconfig.VolumeConfiguration{
+				EnableDynamicProvisioning:  true,
 				EnableHostPathProvisioning: false,
+				FlexVolumePluginDir:        "/usr/libexec/kubernetes/kubelet-plugins/volume/exec/",
```

Comment:
This is the directory in which admins should put their binaries if they choose to use FlexVolumes.  The plugin itself will list/read whatever binaries it finds in that directory and invoke them by name as found in the FlexVolume API.

It is considered an "alpha" feature upstream. I'm not sure if Openshift officially supports it or not or how extensively it's been tested downstream.

https://github.com/kubernetes/kubernetes/blob/master/examples/flexvolume/README.md